### ISSUE TITLE
JAWS changes columns when scrolling through table that updates

### ISSUE THREAD

**Issue by jdrush89** [FreedomScientific/VFO-standards-support#608]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Start JAWS
> 2. Go to https://codesandbox.io/s/intelligent-resonance-ujiymw
> 3. Press 't' to put virtual focus on the table
> 4. Press ctrl +alt + right arrow to select the second column
> 5. Hold ctlr + alt + down to navigate through the rows

### Expected result

> The virtual focus should stay on the second column as rows are navigated

### Actual result

> The virtual focus sometimes jumps to a different column

### Example

> https://codesandbox.io/s/intelligent-resonance-ujiymw

### Additional Information

May need to scroll up and down the table a couple times to see the bug repro.  The table is virtualized, so new elements come in as it is scrolled.  It seems like the column switching is due to the table being modified.  This should be supported though, other screen readers like NVDA and Narrator handle this correctly.

#### JAWS version and build number

Professional Edition Version 2022.2110.36 ILM
 
#### Operating System and version

Windows 10 Enterprise Build 19041.vb_release.191206-1406

#### Browser and version:

Chrome Version 99.0.4844.51 (Official Build) (32-bit)
